style.css

```diff
@@ -425,6 +425,92 @@ body {
   .innerSkills li {
     padding: 5px;
   }
+
+  /* contact form and footer section  */
+
+  .formContainer {
+    padding-top: 5rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .formContainer h2 {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    text-align: center;
+    color: #172b4d;
+    padding: 0 24px;
+  }
+
+  .contactForm {
+    margin-top: 3rem;
+    display: flex;
+    padding: 0 24px;
+    flex-direction: column;
+    row-gap: 45px;
+  }
```

Comment:
[Optional] The styling applied to your contact form is causing a horizontal scroll on your webpage. I highly suggest you take a look at this and fix it so your project will be perfect😎